### ISSUE TITLE
Forced snatch of a multi ep doesnt mark as snatch

### ISSUE THREAD

**Issue by duramato** [pymedusa/Medusa#229]
Branch/Commit: 134830fc282a413696af121bb04af20640401e88
OS: Ubuntu 14
What you did: For the snatch from manual snacth to mark all ep as snatched
What happened: When manually snatching `X-Files.S01E01-17.DVDRip.XviD-SDG` only the ep the manual snatch page was got marked as snatched
What you expected: All of the ep to be marked as snatched as on a manual search

**Comment by duramato** [comment]
This is happening again:

For: `The.Path.S02e07-13.mux.h264.ita.Ac3.webmux.-.ade`

Only the episode 11 got marked as snatched: 

![image](https://user-images.githubusercontent.com/10121067/30002560-315c65bc-90a4-11e7-994f-4b36e2bf6519.png)


```
For: The.Path.S02e07-13.mux.h264.ita.Ac3.webmux.-.ade
GuessIt found: {
    "title": "The Path",
    "season": 2,
    "episode": [
        7,
        8,
        9,
        10,
        11,
        12,
        13
    ],
    "episode_title": "mux",
    "video_codec": "h264",
    "language": "Italian",
    "audio_codec": "AC3",
    "format": "WEB-DL",
    "other": "Mux",
    "release_group": "ade",
    "type": "episode"
}
```
@fernandog

**Comment by medariox** [comment]
Just tested this and it is working as expected now.

**Comment by duramato** [comment]
Just tested on a clean develop install and it still doesn't. For: `12.Monkeys.S04E10-E11.720p.HDTV.x264-KILLERS`

Log: https://gist.github.com/duramato/7a3bf8aedc65eb44a6b9b1fc76490b70

Only E11 get's marked as snatched.

![image](https://pls.gtfo.accountant/rX5VO.png)

**Comment by medariox** [comment]
That's not a manual snatch?

**Comment by duramato** [comment]
Yes it seems the issue is now only present on a forced search

**Comment by sharkykh** [comment]
Fixed in develop with #4675.
